sympy/solvers/solveset.py

```diff
@@ -63,18 +64,19 @@ def _invert(f_x, y, x, domain=S.Complexes):
     >>> from sympy.solvers.solveset import invert_complex, invert_real
     >>> from sympy.abc import x, y
     >>> from sympy import exp, log
+    >>> from sympy import exp, sin, Symbol, pprint, S
 
     When does exp(x) == y?
 
-    >>> invert_complex(exp(x), y, x)
-    (x, ImageSet(Lambda(_n, I*(2*_n*pi + arg(y)) + log(Abs(y))), Integers()))
+    >>> pprint(invert_complex(exp(x), y, x), use_unicode = False)
```

Comment:
Pretty printing is not needed, I suppose.
These changes make the doc strings look irregular (with some results being pretty-printed while others display the returned values from the function calls)
Can you share the particular reason why you have made this change ?

@asmeurer @hargup @aktech :  What's the convention for writing examples in docstrings ?